documentation/src/main/asciidoc/userguide/chapters/domain/inheritance.adoc

```diff
@@ -313,24 +313,6 @@ include::{extrasdir}/entity-inheritance-table-per-class-query-example.sql[]
 ====
 Polymorphic queries require multiple UNION queries, so be aware of the performance implications of a large class hierarchy.
 
-Unfortunately, not all database systems support UNION ALL, in which case, UNION is going to be used instead of UNION ALL.
-
-The following Hibernate dialects support UNION ALL:
-
-- `AbstractHANADialect`
-- `AbstractTransactSQLDialect`
-- `CUBRIDDialect`
-- `DB2Dialect`
-- `H2Dialect`
-- `HSQLDialect`
-- `Ingres9Dialect`
-- `MySQL5Dialect`
-- `Oracle8iDialect`
-- `Oracle9Dialect`
-- `PostgreSQL81Dialect`
-- `RDMSOS2200Dialect`
-====
-
```

Comment:
Was that on purpose? It seems unrelated and breaks formatting (you're not closing the admonition block).

Reply:
I'll fix the block closing, but the section is unnecessary, because all supported dialect versions support `union all` now.